default/scripting/ship_parts/Bays.focs.txt

```diff
@@ -9,4 +9,15 @@ Part
     location = OwnedBy empire = Source.Owner
     icon = "icons/ship_parts/fighters-1.png"
 
+Part
+    name = "FT_BAY_2"
+    description = "FT_BAY_1_DESC"
+    class = FighterBay
+    capacity = 3
```

Comment:
Perhaps not, and after reviewing this and your comment about unlocking three hangars at once, I'm considering taking a different approach. As they are now, fighters don't increase PP cost nearly as much as direct fire weapons, having a second bay part was an attempt to address that. A better solution might be a single bay part and 5 hangar parts, one for each kind of direct fire weapon plus the decoys, all of which upgrade capacity via their own line of techs.